### PR TITLE
updated german translations, fixed capitalisation mistake

### PR DESCRIPTION
**Pull request by gamersi**

### CODE REVIEW

File: shared/plugins/i18n/locales/de.json [hunk 1]
```diff
@@ -219,15 +221,24 @@
         "favorite": "Sie werden {modelName} aus Ihren Favoriten löschen.",
         "bookmark": "Sie werden {modelName} aus Ihren Lesezeichen löschen.",
         "post": "Sie werden diesen Post löschen.",
-        "community": "Sie werden diese Community löschen."
+        "community": "Sie werden diese Community löschen.",
+        "account": "Sie werden Ihren Account löschen"
```
Comment (staniel359) [inline shared/plugins/i18n/locales/de.json]: @gamersi Missed the dot.

File: shared/plugins/i18n/locales/de.json [hunk 2]
```diff
@@ -260,7 +271,7 @@
       "remember": "Eingeloggt bleiben",
       "asCommunity": "Als Community"
     },
-    "extra": "Zusätzliche infos",
+    "extra": "zusätzliche Infos",
```
Comment (staniel359) [inline shared/plugins/i18n/locales/de.json]: @gamersi Shouldn't this be capitalized?

**Review by staniel359 — CHANGES_REQUESTED** [review]

File: shared/plugins/i18n/locales/de.json (hunk 2 above)
Comment (gamersi) [reply shared/plugins/i18n/locales/de.json]: I dont know the context. if it is in a sentence zusätzliche is not capitalized. if it is standalone zusätzliche is capitalized

Comment (staniel359) [reply shared/plugins/i18n/locales/de.json]: @gamersi Then please refer to `en.json`. It's capitalized there.

Comment (gamersi) [reply shared/plugins/i18n/locales/de.json]: changed it in the newest commit.

**Review by staniel359 — APPROVED** [review]
@gamersi Good, thank you.

**Pull request merged**